support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -15,6 +15,18 @@ Next:
     copy_executable: {{ copy_executable | default("None") }}
     submit_script_template: {{ submit_script_template | default("None") }}
     submit: True
+{% elif eccentricity_control %}
+Next:
+  Run: spectre.Pipelines.Bbh.EccentricityControlMinimal:ecc_control_fit
+  With:
+    h5_file: ./BbhReductions.h5
+    subfile_name_aha: ApparentHorizons/ControlSystemAhA_Centers.dat
+    subfile_name_ahb: ApparentHorizons/ControlSystemAhB_Centers.dat
+    tmin: 3000
+    tmax: 3800
+    angular_velocity_from_xcts: 0.014
+    expansion_from_xcts: -1
+    output: ecc_control.pdf
```

Comment:
Add `id_input_file_path: {{ id_input_file_path }}` here. Then in `start_inspiral()` where you pass `eccentricity_control` boolean to `schedule()` you also pass on `id_input_file_path`.